Example/Tests/HaventecAuthenticateTests.swift

```diff
@@ -27,273 +27,220 @@ class HaventecAuthenticateTest: XCTestCase {
 
     override func setUp() {
         // Put setup code here. This method is called before the invocation of each test method in the class.
-
-        destroyTestKeychainData()
-
-        do {
-            try HaventecAuthenticate.initialiseStorage(username: "")
-        } catch {
-            XCTFail()
-        }
+        KeychainWrapper.standard.removeAllKeys()
```

Comment:
This doesn't make sense to me. The state of the keychain should be empty before and after each test case, otherwise it wouldn't be a unit test if there were dangling keys.

Reply:
Actioned